content/rest-api/applications.md

```diff
@@ -137,3 +137,126 @@ curl 'https://api.codemagic.io/apps/your-app-id/encrypt-environment-variable' \
 ```yaml
 {"encrypted": "Encrypted(Z0FBQUFBQmZMVkhwb3Q3QlJtRlVOeVFJcEJvTTRtWnZablpqMS0xN2V6dllTell1ODZSd2FUcnNqMUlZT09QY1paV0pjbVRfUlVJeDUxRWIzX1paOEZlc1dSdi1XMXlkUFVIdjNIZ2VqcE5Ja0tpMjlPWjhlSTQ9)"}
 ```
+
+## Modify application variables and secrets
+
+Codemagic allows you to fetch and modify application variables and secrets using the REST API. Note that these variables are available only for the applications that are configured using the `codemagic.yaml` file. For such applications, variables and secrets are manually configured on the **Environment variables** tab in your application settings. These variables and secrets can be accessed in your configuration file with the use of [groups](../building/environment-variable-groups).
+
+### Fetch variables
+
+`GET /apps/:id/variables`
+
+#### CURL request

```

Comment:
try to keep the same structure for all new actions. I'd recommend using:
- Parameters
- Example
- Response